.gitignore

```diff
@@ -15,8 +15,6 @@ tdrs-frontend/.env*
 
 
 #BACKEND
-tdrs-backend/*/env_vars/*.env*
```

Comment:
.env files are required for tests to run - will review this strategy down the road